File: problem_twelve.js
// Probability 
const Fruits =["apple","banana","blackcurrant","mango","apple","apple","banana","mango","mango","apple"]
const input_fruit = "apple"
function frauitsFrobability(list,item){
    var list_len = list.length
    var count = 0
    for(var i = 0; i< list_len;i++){
        if (list[i]== item){
            count+=1
        }
    }
    var proba = count / list_len
    return proba
}
console.log(frauitsFrobability(Fruits,input_fruit))

File: problem_twelve.py
# probability
Fruits =["apple","banana","blackcurrant","mango","apple","apple","banana","mango","mango","apple"]
data = input("Enter probability fruite name:")
def Fruits_prbability(list,fruit):
    list_len =len(list)
    count = 0
    for i in list:
        if i == fruit:
            count+=1
    probability = count/ list_len        
    return probability       
print(Fruits_prbability(Fruits,data))



